gamestonk_terminal/stocks/fundamental_analysis/dcf_model.py

```diff
@@ -0,0 +1,841 @@
+    other_args : List[str]
+        argparse other args
+    ticker : str
+        Fundamental analysis ticker symbol
+    """
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        prog="dcf",
+        description="""
+            Generates a completed discounted cash flow statement. The statement uses machine
+             learning to predict the future financial statement, and then predicts the future
+             value of the stock based on the predicted financials.""",
+    )
+    parser.add_argument(
+        "-a",
+        "--audit",
+        action="store_true",
+        dest="audit",
+        default=False,
+        help="Confirms that the numbers provided are accurate.",
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if not ns_parser:
+            return
+
+        dcf_view = CreateExcelFA(ticker, ns_parser.audit)
+        trypath = dcf_view.create_workbook()
+        print(
+            f"Analysis successfully ran for {ticker}\nPlease look in {trypath} for the file."
+        )
+
+    except Exception as e:
+        print(e, "\n")
+
+
+class CreateExcelFA:
+
+    # pylint: disable=R0902
+    # pylint: disable=R0912
+
+    def __init__(self, ticker: str, audit: bool):
+        self.audit: bool = audit
+        self.wb: Workbook = Workbook()
+        self.ws1: worksheet = self.wb.active
+        self.ws2: worksheet = self.wb.create_sheet("Free Cash Flows")
+        self.ws3: worksheet = self.wb.create_sheet("Explanations")
+        self.ws1.title = "Financials"
+        self.ticker: str = ticker
+        self.now: str = datetime.now().strftime("%m/%d/%Y, %H:%M:%S").replace("/", "-")
+        self.letter: int = 0
+        self.is_start: int = 4
+        self.bs_start: int = 18
+        self.cf_start: int = 47
+        self.len_data: int = 0
+        self.len_pred: int = 10
+        self.years: List[str] = []
+        self.rounding: int = 0
+        self.df_bs: pd.DataFrame = self.get_data("BS", self.bs_start, False)
+        self.df_is: pd.DataFrame = self.get_data("IS", self.is_start, True)
+        self.df_cf: pd.DataFrame = self.get_data("CF", self.cf_start, False)
+        self.info = yf.Ticker(ticker).info
+
+    def create_workbook(self):
+        self.ws1.column_dimensions["A"].width = 25
+        self.ws2.column_dimensions["A"].width = 22
+        for column in var.letters[1:21]:
+            self.ws1.column_dimensions[column].width = 14
+        for column in var.letters[1:21]:
+            self.ws2.column_dimensions[column].width = 14
+
+        self.ws3.column_dimensions["A"].width = 3
+        self.create_header(self.ws1)
+        self.create_header(self.ws2)
+        self.create_header(self.ws3)
+        self.add_estimates()
+        self.create_dcf()
+        if self.audit:
+            self.run_audit()
+
+        trypath = f"../GamestonkTerminal/exports/excel/{self.ticker}-{self.now}.xlsx"
```

Comment:
So that this is cross platform instead of using "/" we should use os.path.join.

E.g.
```
trypath =os.path.join("..",  "GamestonkTerminal", "exports", "excel", f"{self.ticker}-{self.now}.xlsx")
```

Reply:
So that the files get sorted by ticker and also by date, can we have the name of the file to be:

```
AAPL 2021/08/14 10:20:30
```

So that the order is from big to small.